src/transformState.js

```diff
@@ -6,6 +6,31 @@
  */
 function transformState(state, actions) {
   // write code here
+  for (const action of actions) {
+    const { type, extraData, keysToRemove } = action;
+
+    switch (type) {
+      case 'addProperties':
+        Object.assign(state, extraData);
+
+        break;
+
+      case 'removeProperties':
+        keysToRemove.forEach((key) => delete state[key]);
+
+        break;
+
+      case 'clear':
+        Object.keys(state).forEach((key) => delete state[key]);
```

Comment:
it will take less operations
```suggestion
        for (const key in state) {
          delete state[key];
        }
```